mysql-test/suite/rocksdb_rpl/r/rpl_rocksdb_row_auto_increment.result

```diff
@@ -321,6 +321,8 @@ DROP TABLE t2;
 include/sync_slave_sql_with_master.inc
 include/stop_slave.inc
 RESET SLAVE;
+Warnings:
```

Comment:
Here and everywhere in this file
Probably it's better to change the command in the .test file

Reply:
Explained.